### PR TITLE
links SkopeRules to existing literature

### PR DESCRIPTION
**Pull request by ngoix**
**What does this implement/fix?**
- add references in README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,12 +29,29 @@ class, i.e. detecting with high precision instances of this class.
...
-------------
+The main advantage of decision rules is that they are offering interpretable models. The problem of generating such rules has been widely considered in machine learning, see e.g. RuleFit [1], Slipper [2], LRI [3], MLRules[4].
+
+A decision rule is a logical expression of the form "IF conditions THEN reponse". In a binary classification setting, if an instance satisfies conditions of the rule, then it is assigned to one of the two classes. If this instance does not satisfy conditions, it remains unassigned.
+
+1) In [2, 3, 4], rules induction is done by considering each single decision rule as a base classifier in an ensemble, which is built by greedily minimizing some loss function.
+
+2) In [1], rules are extracted from an ensemble of trees; a weighted combination of these rules is then built by solving a L1-regularized optimization problem over the weights as described in [5].
+
+In this package, we use approach 2). Rules are extracting from tree ensemble, which allow us to take advantage of existing fast algorithms to produce such tree ensemble. Too similar or duplicated rules are then removed.
```
Comment (datajms) [inline README.md]: Rules are extractED from ...

existing fast algorithms (such as bagged decision trees, or gradient boosting)

Comment (datajms) [inline README.md]: Too similar or duplicated rules are then removed, based on a similarity threshold of their supports.

**Comment by datajms** [comment]
Any idea of what happened to Travis?

**Comment by ngoix** [comment]
No idea... Maybe it is in a bad day. Will try to figure out what happened.

**Comment by datajms** [comment]
Apart from my comments, LGTM. +1

**Comment by ngoix** [comment]
I relaunched the test, all checks passed. Merging!

**Pull request merged**